ruby/.rubocop.yml

```
@@ -31,11 +31,11 @@ Layout/LineLength:
   Enabled: true
   Max: 120
   Exclude:
-    - spec/**/*
+    - !ruby/regexp /spec\/(?!components\/previews)/
```

wdyt about extracting this into a variable so people without our context can still understand why it's here.
```suggestion
    - $(spec_dir_except_component_previews)
```

---

It's YAML. I'd like to keep it that way.